hardware/benchmark/disk.py

```diff
@@ -89,6 +93,8 @@ def run_fio(hw_lst, disks_list, mode, io_size, time, rampup_time):
                                shell=True, stdout=subprocess.PIPE)
     current_disk = ''
     for line in fio_cmd.stdout:
+        if isinstance(line, six.binary_type):
+            line = line.decode()
```

Comment:
Similarly to another PR: should we pass `errors="ignore"` here?

Reply:
I was thinking for the other PR to merge and just call the function from detect_utils
it's the same issue